frontend/src/models/backend.ts

```diff
@@ -22,6 +22,8 @@ type IBackendState = { [key: string]: any } & {
     isElevated: boolean
     privateIP: ipAddress
     hostname: string
+    oobAvailable: boolean
+    oobActive: boolean
```

Comment:
I think these should be moved out of the env so that they can be called, especially if they are called at an interval, independently from the environment vars. Or maybe just don't call at an interval.

Reply:
moved to lan